board/embeddedartists/mx7ulpea-ucom/mx7ulpea-ucom.c

```diff
@@ -99,6 +99,18 @@ int board_init(void)
 	return 0;
 }
 
+/* will reset the M4 and the A7 */
+void board_m4_restart(void)
```

Comment:
Technically this should only apply to Aeler's HW (Atlas).  Not the EA 7ulp SoM.  We don't have a split out DTS file for that, but let me double-check if pulling PTC10 on the EA HW will cause problems.

Reply:
This one is ok for now. Looks like PTC10 isn't being used by default on EA board (it's connected to J39 PIN47 labelled: TP_RST-GPIO).  U-boot pulling this shouldn't matter.
We may consider breaking out Aeler HW DTS at some point.